### PR TITLE
Opal-427

### PR DESCRIPTION
**Pull request by CW-USAF**

### CODE REVIEW

**Comment by CW-USAF** [comment]
The metadata_db.py file was requested by Jon to keep the original functionality of mongo. I think it is used in some ingestion scripts so this makes it so we don't have to change that functionality.

File: weave/__init__.py [hunk 1]
```diff
@@ -24,15 +24,15 @@
...
+__version__ = "1.7.0"
 
 __all__ = [
     "Basket",
     "IndexPandas",
     "IndexSQLite",
     "IndexSQL",
     "Pantry",
-    "load_mongo",
+    "MongoDB"
```
Comment (JA-USAF) [inline weave/__init__.py:37]: Keep "load_mongo" as well to keep existing code from breaking on imports

**Review by hh-usaf — APPROVED** [review]
Code looks good and passed local pytests

**Review by hh-usaf — APPROVED** [review]

**Review by JA-USAF — APPROVED** [review]
🐐

**Pull request merged**